akka-cluster/src/multi-jvm/scala/akka/cluster/MultiDcSplitBrainSpec.scala

```diff
@@ -110,7 +141,6 @@ abstract class MultiDcSplitBrainSpec
 
     "be able to have data center member leave while there is inter data center split" in within(20.seconds) {
       splitDataCenters(dc1, dc2)
```

Comment:
This one fails now, because first and second (and possibly more) does not get an `UnreachableDataCenter` event.